### PR TITLE
get_remote_version() needs to run in a docker

### PR DESCRIPTION
**Pull request by rafaelvanoni**

### CODE REVIEW

**Comment by caseydavenport** [comment]
@rafaelvanoni could you provide a bit more context? Why does this command need to run in a container?

**Comment by rafaelvanoni** [comment]
> @rafaelvanoni could you provide a bit more context? Why does this command need to run in a container?

We need the `$(GIT_CONFIG_SSH)` bit for private repos.

File: Makefile.common [hunk 1]
```diff
@@ -218,15 +218,15 @@ DOCKER_GO_BUILD := $(DOCKER_RUN) $(CALICO_BUILD)
...
+	$(shell $(DOCKER_RUN) $(CALICO_BUILD) sh -c '$(GIT_CONFIG_SSH) git ls-remote https://$(1) $(2) | cut -f1')
 endef
 
 # update_pin updates the given package's version to the latest available in the specified repo and branch.
 # $(1) should be the name of the package, $(2) and $(3) the repository and branch from which to update it.
 define update_pin
 	$(eval new_ver := $(call get_remote_version,$(2),$(3)))
 
-	$(DOCKER_RUN) $(CALICO_BUILD) sh -c '\
+	$(DOCKER_RUN) -i $(CALICO_BUILD) sh -c '\
```
Comment (caseydavenport) [inline Makefile.common]: What's the `-i` needed for here?

Comment (rafaelvanoni) [reply Makefile.common]: So we can `ctrl-c` the call if needed.

**Pull request merged**